iac-extensions/kubernetes/src/test/resources/checks/MemoryLimitCheck/limit_range_cpu.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
```

Comment:
Actually, you can put all `LimitRange`s in one file and use different namespaces. 
Other cases to verify:
* `type: Pod` 
* `type: PersistentVolumeClaim` 
* `type: with-global-limit.example` - Full qualified name - it will be FN for now
* `min` instead of `default` is defined

Reply:
Yeah, I'll add more tests later. There's a bit of a problem with `verifyContent` and x-file Kubernetes/Helm, which makes it not so straightforward to write many tests with different namespaces